src/data/nat/basic.lean

```diff
@@ -110,6 +109,9 @@ instance nat.subtype.semilattice_sup_bot (s : set ℕ) [decidable_pred (∈ s)]
   ..subtype.linear_order s,
   ..lattice_of_linear_order }
 
+lemma nat.subtype.semilattice_sup_bot_bot_apply {s : set ℕ} [decidable_pred (∈ s)]
+  [h : nonempty s] : ((⊥ : s) : ℕ) = nat.find (nonempty_subtype.1 h) := rfl
+
```

Comment:
Where did this lemma come from? It's not mentioned in the PR description.

Reply:
ahh crap, this was meant to be for another PR and got moved in the stash (branch#count)

Reply:
bors r-

Reply:
fixed now!

Reply:
Huh, I guess bors ignored that r-. Didn't matter in the end.